app/build.gradle

```diff
@@ -327,9 +327,8 @@ dependencies {
     implementation 'com.google.android.gms:play-services-location:10.2.1'
     implementation "com.google.android.gms:play-services-gcm:10.2.1"
     implementation 'com.squareup.wire:wire-runtime:2.2.0'
-    implementation 'com.squareup.okhttp:okhttp:2.7.5'
```

Comment:
I think this line should be restored otherwise okhttp 2.5 will be used in the dependency instead of 2.7.5

Reply:
I don't know why it should use v2.5 if no dependency for v2.x is declared at all. I got no compilation errors. Anyway, I'll revert this change in favour of #1812.